images/kernel/upgrade-config.sh

```diff
@@ -21,7 +21,13 @@ if [[ ${FROM} != ${TO} ]]; then
     cp ${FROM} ${TO}
 fi
 
-docker run -it \
+if ! [ -x "$(command -v podman)" ]; then
+  oci_tool=podman
+else
+  oci_tool=docker
```

Comment:
Thanks for the patch :wave:

Would `DOCKER=podman make build` be sufficient for you?
This would be consistent with the main `Makefile` we already have.

Other usage would be:
`DOCKER="sudo docker" make build`

The kernel build helper scripts would also need to be modified to use this ENV var to override docker.
We could do a followup PR if you don't have the bandwidth to do so.

Reply:
No worries: `DOCKER=podman`.  Will try get to that this week and force update this.

Reply:
thanks!